components/SpotDetails/SpotSocial/SpotSocial.js

```diff
@@ -1,24 +1,57 @@
 import SpotSocialComment from "./SpotSocialComment";
 import SpotSocialForm from "./SpotSocialForm";
-import { useState } from "react";
 import styled from "styled-components";
 
-export default function SpotSocial({ ID }) {
-  const [comments, setComments] = useState([]);
-
+export default function SpotSocial({ SpotData }) {
   function handleNewComment(comment) {
```

Comment:
please use camel case for all "normal" JS data types (exception: React components and Mongoose models).
`SpotData` => `spotData`